google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/GoogleCloudApiSelectorPanel.java

```diff
@@ -183,21 +185,26 @@ private void createUIComponents() {
               if (!model.isSelectionEmpty()) {
                 int selectedIndex = model.getMinSelectionIndex();
                 CloudLibrary library =
-                    (CloudLibrary) cloudLibrariesTable.getModel().getValueAt(selectedIndex, 0);
-                boolean isSelected =
-                    (boolean) cloudLibrariesTable.getModel().getValueAt(selectedIndex, 1);
+                    (CloudLibrary)
+                        cloudLibrariesTable.getModel().getValueAt(selectedIndex, CLOUD_LIBRARY_COL);
+                boolean addLibrary =
+                    (boolean)
+                        cloudLibrariesTable
+                            .getModel()
+                            .getValueAt(selectedIndex, CLOUD_LIBRARY_SELECT_COL);
```

Comment:
Looks like an identical code here (190:194) and in lines 204:207, could we use a common method instead, something like `updateManagementUI()`?